html/rendering/non-replaced-elements/form-controls/button-style.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<title>default style on buttons</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```

Comment:
Could you link to the HTML spec PR from here? Or just adding links when it gets merged into the spec I guess.

Reply:
The relevant section of the spec is supposed to be able to get from the last dir name for html, I think. Is that enough?

Reply:
Ok, that works, thanks :)